Appraisals

```diff
@@ -1,17 +1,16 @@
 appraise 'rails-4-2' do
   gem 'rails', '~> 4.2.0'
+  gem 'activerecord-jdbcsqlite3-adapter', '~> 1.3.24', platforms: :jruby
 end
 
 appraise 'rails-5-0' do
   gem 'rails', '~> 5.0.0'
   gem 'rack'
   gem 'rails-controller-testing'
-  gem 'activerecord-jdbcsqlite3-adapter', github: 'jruby/activerecord-jdbc-adapter', branch: 'rails-5', platforms: :jruby
 end
 
 appraise 'rails-5-1' do
   gem 'rails', '~> 5.1.0'
   gem 'rack'
   gem 'rails-controller-testing'
-  gem 'activerecord-jdbcsqlite3-adapter', github: 'jruby/activerecord-jdbc-adapter', branch: 'rails-5', platforms: :jruby
```

Comment:
@smudge why is it that we can remove this from here? we don't need this on rails 5+?

Reply:
oh, i'm guessing that it doesn't work on rails 5+, correct?

Reply:
Nope, I'm still confused.

Reply:
I've left a generic line in the`Gemfile`, which the other appraisals will inherit:

```
gem 'activerecord-jdbcsqlite3-adapter', platforms: :jruby
```